Library/Subscription.cs

```diff
@@ -341,6 +345,29 @@ public void Postpone(DateTime nextRenewalDate)
                 ReadXml);
         }
 
+        public bool UpdateNotes(String customerNotes, String termsAndConditions, String vatReverseChargeNotes)
+        {
+
+            if (customerNotes != null)
+                CustomerNotes = customerNotes;
```

Comment:
Maybe we can pass in the notes to the `WriteSubscriptionNotesXml` method and then if the save is successful assign them to the subscription